locksmith/__tests__/controllers/v2/keyController.test.ts

```diff
@@ -0,0 +1,314 @@
+import request from 'supertest'
+import { loginRandomUser } from '../../test-helpers/utils'
+
+const app = require('../../../src/app')
+
+jest.setTimeout(600000)
```

Comment:
Any reason why you need this? It's usually an indication that there are network calls happening and there should not be any :/